src/clr-angular/emphasis/alert/alert.html

```diff
@@ -15,7 +15,13 @@
     <div class="alert-items">
         <ng-content></ng-content>
     </div>
-    <button type="button" class="close" *ngIf="closable" (click)="close()">
+    <button 
+        type="button" 
+        class="close" 
+        *ngIf="closable" 
+        (click)="close()" 
+        [attr.aria-label]="clrCloseButtonAriaLabel"
+        >
         <clr-icon shape="close" [attr.title]="commonStrings.close"></clr-icon>
```

Comment:
I don't think this should hold up the PR. But do we need the `attr.title` inside of a button with an `aria-label` on it?